docs/atm10/oredistro.md

```diff
@@ -1,73 +1,79 @@
 ---
 title: Mining Dimension Ore Distribution
 description: All ore spawn levels in the Mining Dimension!
-authors: 
- - Vidal (Original)
- - ArcTrooper (Editor)
+authors:
+    - Vidal (Original)
+    - ArcTrooper (Editor)
 ---
 
 # Mining Dimension Ore Distribution
 
 [Vidal's Original Spreadsheet.](https://docs.google.com/spreadsheets/d/1WA9Ek1k17u6YA9JJBSayUL1x_D8OMp134oQeVXBftDU/edit?gid=0#gid=0)
 
 ???+ warning "Max/Min Y-Level Sort Broken"
-    The sorting for the y-levels is broken for numbers currently. Alphabetical sorting is the only one that works right now. I will fix sorting by numbers in the future.
```

Comment:
This indent needs to be there for the admonition to work

Reply:
Sorry about that - my linter "fixed" it.  will resubmit PR momentarily.